scripts/generate_book.py

```diff
@@ -91,6 +91,8 @@ def _case_sensitive_fs(path):
             with open(fname, 'wt') as fobj:
                 fobj.write('text')
         written = glob(root + '*')
+    except Exception:
+        written = []
```

Comment:
Thanks for the catch.  Do you think the routine should return False in this case, or raise an error?

Reply:
Do you know why this would error on some machines? Actually it does think that we'd want an error thrown here

Reply:
I guess when the BUILD_FOLDER directory is not writable?  Maybe it does not exist yet?  I guess I could create it, if it does not exist?

Yes, I think an error is the right thing too.

Reply:
cool - would you like to wrap this one up, or want me to push to the branch again? No hurry :-)